lib/twilioconnection.js

```diff
@@ -205,7 +206,7 @@ class TwilioConnection extends StateMachine {
           // Do nothing.
           break;
         case 'heartbeat':
-          this._handleHeartbeat();
+          // Do nothing.
```

Comment:
@makarandp0 You can remove this case altogether I think.

Reply:
I think keeping this case with `// do nothing` comment helps with understaning possible `message.type`  values.